tests/test_web_scraper.py

```diff
@@ -1,21 +1,19 @@
 import unittest
 
-from selenium import webdriver
-
-from src.modules.web_scraper import open_page, check_header_present
+from src.modules.web_scraper import open_page, check_header_present, get_last_reviewed_date, get_last_tested_date,get_prepared_date
 
 
 class TestWebScraper(unittest.TestCase):
 
     def setUp(self):
-        self.driver = open_page("https://www.example.com", True)
+        self.driver = open_page("https://www.nmsw.homeoffice.gov.uk/accessibility-statement", True)
 
     def tearDown(self):
         self.driver.quit()
 
     def test_scrape_page(self):
         title = self.driver.title
-        self.assertEqual(title, "Example Domain")
+        self.assertEqual(title, "Accessibility statement for National Maritime Single Window")
 
     def test_check_header_present(self):
         actual = check_header_present(self.driver, "Example Domain")
```

Comment:
Will need to update this test if we're changing the URL

Reply:
Done